Yosemite/Yosemite/Stores/AppSettingsStore.swift

```diff
@@ -8,12 +8,15 @@ public class AppSettingsStore: Store {
     ///
     private let fileStorage: FileStorage
 
+    private let generalAppSettings: GeneralAppSettingsStorage
+
     /// Designated initaliser
     ///
     public init(dispatcher: Dispatcher,
                 storageManager: StorageManagerType,
                 fileStorage: FileStorage) {
         self.fileStorage = fileStorage
+        self.generalAppSettings = GeneralAppSettingsStorage(fileStorage: fileStorage)
```

Comment:
nit: is it possible to DI `GeneralAppSettingsStorage` using the instance from `ServiceLocator` during initialization of the `AppSettingsStore`?

Reply:
Yes, I think that makes sense. I guess I skipped it because right now we don't really need to have a single instance of `GeneralAppSettingsStorage`, but it seems more future-proof to do it that way.